src/Project/Project.js

```diff
@@ -4,33 +4,48 @@ import './Project.scss';
 import trashBin from '../images/rubbish.png';
 import Palette from '../Palette/Palette';
 
-const Project = ({projects}) => {
+const Project = ({name, id, palettes}) => {
 
   // add removeProject or pass as props from ProjectContainer
   // if created here import apicall, usedispatch and action
 
-   const palettes = useSelector(state => state.palettes)
+  //  const actualPalettes = useSelector(state => state.palettes);
 
-    const displayPalettes = palettes
-      ? 
-      palettes.map(palette => (
-      <Palette {...palette} key={palette.id} />))
-      : '';
+    const displayPalettes = () => {
+      // return palettes ? palettes.filter(palette => { //returns true or false;
+      //   console.log('palette', palette)
+      //   const projectNames = projects.map(project => {
+      //     console.log('project', project)
+      //     if(palette.project_id === project.id) {
+      //       return <Palette {...palette} key={palette.id}/>
+      //     }
+      //   })
+      //   return projectNames;
+      //   // console.log('MAP', MAP)
+      // }) //map then convert to palette objects
+      //   : '';
 
-    const displayNames = projects.map(project => {
-      return project.project_name
-    });
-    
+      return palettes.map(palette => {
+        console.log('chicken', palette)
+        return <Palette {...palette} key={palette.id} />
+
+      })
+      
+
+    } 
+      
     return(
       <>
-        <h3 className="project_name">{displayNames}</h3>
-          {displayPalettes}
+        <h3 className="project_name">{name}}</h3>
+          {/* {displayPalettes} */}
```

Comment:
On line 39 there is an extra '}' after name. Also, this should be passed in as palette_name to show up.

Reply:
good lookout, I changed it.

Reply:
yup saw this. Thank you.